api/src/scripts/populate_db_gbfs.py

```diff
@@ -0,0 +1,147 @@
+from datetime import datetime
+
+import pandas as pd
+import pytz
+
+from database.database import generate_unique_id, configure_polymorphic_mappers
+from database_gen.sqlacodegen_models import Gbfsfeed, Location, Gbfsversion, Externalid
+from scripts.gbfs_utils.comparison import generate_system_csv_from_db, compare_db_to_csv
+from scripts.gbfs_utils.fetching import fetch_data, get_data_content, get_gbfs_versions
+from scripts.gbfs_utils.license import get_license_url
+from scripts.populate_db import DatabasePopulateHelper, set_up_configs
+
+OFFICIAL_VERSIONS = [
```

Comment:
I'm not sure about the best approach for this, but this list needs to be more visible and easy to spot when a new GBFS version is released. Without over-architecting a complex solution, adding a documentation section to the README file in this repository may help users find the list when needed.

Reply:
Not sure the readme of our repo is the appropriate place for this information taking into account that we are not serving gbfs feeds through the api yet